Jenkinsfile

```diff
@@ -0,0 +1 @@
+buildPlugin()
```

Comment:
We also want to collect findbugs and checkstyle reports. And mark the build unstable if there are any checkstyle violations.
See [Gerrit Trigger](https://github.com/jenkinsci/gerrit-trigger-plugin/blob/master/Jenkinsfile) as an example.

Reply:
@rsandell TBH, I don't really know, I am coding blind at the moment ;) If I remember well it should be.

Do you need to approve the change in order for Jenkins to start building the updated PR?

My original intent was to disable these tests for Windows platform using something like annotations but I was not able to find an example doing that. My primary language is Python so doing anything in Java is much slower, any hints would be appreciated.

Reply:
In this case it needs to be merged in order to be tested.
I could also push it myself to a separate branch, since I'm a repo collaborator Jenkins should see my commit as trusted.
But in this case I think we'll just merge and see what happens :)